assignments/objects.js

```diff
@@ -19,26 +19,77 @@ let example = {
 
 // Write your intern objects here:
 
+let interns = [
+  {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+},
+{
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  speak: function () {
+    return "Hello, my name is Kennan!"
+  }
+},
+{
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+},
+{
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+},
+{
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F",
+  multiplyNums: function(x,y){
+    return (x*y)
+  }
+}]
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(interns[0].name);
+
 // Kennan's ID
 
+console.log(interns[1].id);
+
 // Keven's email
 
+console.log(interns[2].email);
+
 // Gannie's name
 
+console.log(interns[3].name);
+
 // Antonietta's Gender
 
+console.log(interns[4].gender);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+let kennan = interns[1];
+
+console.log(kennan.speak());
 
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
-//console.log(antonietta.multiplyNums(3,4));
+let antonietta = interns[4];
```

Comment:
Here you had to set antonietta because of how you chose to set it up earlier. Just a subtle difference and not anything wrong by any means. Different strokes for different folks.